vnext/Mso/src/dispatchQueue/threadPoolScheduler_win.cpp

```diff
@@ -64,9 +89,15 @@ void ThreadPoolWorkDeleter::operator()(TP_WORK *tpWork) noexcept {
 // ThreadPoolSchedulerWin implementation
 //=============================================================================
 
+std::mutex ThreadPoolSchedulerWin::s_threadPoolWorkMutex;
```

Comment:
>s_threadPoolWorkMutex [](http://example.com/codeflow?start=35&length=21)

(nit) can this match closer to variable name it's protecting:
s_trackedThreadPoolWorkMutex

Reply:
Sorry, the PR is already merged - I put the AutoMerge too early - I will address it in future PR.